src/ontology/edits/oeo-shared.omn

```diff
@@ -2106,25 +2180,33 @@ pull request: https://github.com/OpenEnergyPlatform/ontology/pull/517",
 Class: OEO_00140006
 
     Annotations: 
-        <http://purl.obolibrary.org/obo/IAO_0000115> "Private transport is a transport process in which people use their own vehicle for movement e.g. bicycle, motorcycle and cars, or walking.",
+        <http://purl.obolibrary.org/obo/IAO_0000115> "A private transport is a passenger transport in which people use their own vehicle for movement e.g. bicycle, motorcycle and cars, or walking.",
         <http://purl.obolibrary.org/obo/IAO_0000233> "issue: https://github.com/OpenEnergyPlatform/ontology/issues/503
-pull request: https://github.com/OpenEnergyPlatform/ontology/pull/517",
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/517
+
+Make subclass of 'passenger transport'
+issue: https://github.com/OpenEnergyPlatform/ontology/issues/1055
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1234",
         rdfs:label "private transport"@en
     
     SubClassOf: 
-        OEO_00140003
+        OEO_00010263
     
     
 Class: OEO_00140007
 
     Annotations: 
-        <http://purl.obolibrary.org/obo/IAO_0000115> "Public transport is a transport process in which a number of passengers share a common public transport facility.",
+        <http://purl.obolibrary.org/obo/IAO_0000115> "A public transport is a passenger transport in which a number of passengers share a common public transport facility.",
```

Comment:
why "transport facility" instead of vehicle? What's the difference?

Reply:
yes